src/content/configuration/resolve.md

```diff
@@ -120,7 +120,34 @@ The following table explains other cases:
 
 W> `resolve.alias` takes precedence over other module resolutions.
 
-W> [`null-loader`](https://github.com/webpack-contrib/null-loader) will be deprecated in `webpack@5`. use `alias: { xyz$: false }` or absolute path `alias: {[path.resolve(__dirname, "....")]: false }`
+W> [`null-loader`](https://github.com/webpack-contrib/null-loader) is deprecated in `webpack@5`. use `alias: { xyz$: false }` or absolute path `alias: {[path.resolve(__dirname, "....")]: false }`
+
+W> `[string]` values are supported since webpack 5.0.0.
```

Comment:
It's still in beta, lets just say 5 here too?

Reply:
Done, Let's get this merged, It has been sitting here for almost 6 months now :)